.coverage.Inspiron-3521.21561.440268

```diff
@@ -0,0 +1 @@
s/pandas/tools/plotting.py":[],"/home/sivakar/Projects/pandas/pandas/types/concat.py":[],"/home/sivakar/Projects/pandas/pandas/types/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/types/common.py":[],"/home/sivakar/Projects/pandas/pandas/io/pickle.py":[],"/home/sivakar/Projects/pandas/pandas/io/parsers.py":[],"/home/sivakar/Projects/pandas/pandas/io/s3.py":[],"/home/sivakar/Projects/pandas/pandas/io/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/io/clipboards.py":[],"/home/sivakar/Projects/pandas/pandas/io/date_converters.py":[],"/home/sivakar/Projects/pandas/pandas/io/stata.py":[],"/home/sivakar/Projects/pandas/pandas/io/parquet.py":[],"/home/sivakar/Projects/pandas/pandas/io/pytables.py":[],"/home/sivakar/Projects/pandas/pandas/io/packers.py":[],"/home/sivakar/Projects/pandas/pandas/io/gbq.py":[],"/home/sivakar/Projects/pandas/pandas/io/excel.py":[],"/home/sivakar/Projects/pandas/pandas/io/feather_format.py":[],"/home/sivakar/Projects/pandas/pandas/io/html.py":[],"/home/sivakar/Projects/pandas/pandas/io/common.py":[],"/home/sivakar/Projects/pandas/pandas/io/api.py":[],"/home/sivakar/Projects/pandas/pandas/io/sql.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/latex.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/css.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/printing.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/csvs.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/format.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/style.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/console.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/excel.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/html.py":[],"/home/sivakar/Projects/pandas/pandas/io/formats/terminal.py":[],"/home/sivakar/Projects/pandas/pandas/io/msgpack/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/io/msgpack/_version.py":[],"/home/sivakar/Projects/pandas/pandas/io/msgpack/exceptions.py":[],"/home/sivakar/Projects/pandas/pandas/io/json/json.py":[],"/home/sivakar/Projects/pandas/pandas/io/json/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/io/json/table_schema.py":[],"/home/sivakar/Projects/pandas/pandas/io/json/normalize.py":[],"/home/sivakar/Projects/pandas/pandas/io/sas/sas_xport.py":[],"/home/sivakar/Projects/pandas/pandas/io/sas/sas7bdat.py":[],"/home/sivakar/Projects/pandas/pandas/io/sas/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/io/sas/sas_constants.py":[],"/home/sivakar/Projects/pandas/pandas/io/sas/sasreader.py":[],"/home/sivakar/Projects/pandas/pandas/io/clipboard/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/io/clipboard/clipboards.py":[],"/home/sivakar/Projects/pandas/pandas/io/clipboard/windows.py":[],"/home/sivakar/Projects/pandas/pandas/io/clipboard/exceptions.py":[],"/home/sivakar/Projects/pandas/pandas/_libs/__init__.py":[],"/home/sivakar/Projects/pandas/pandas/_libs/tslibs/__init__.py":[]}}
```

Comment:
Oops. Please be sure not to commit any extraneous files such as this one.

Reply:
Sorry! Did not notice that. Thought `.gitignore` would handle these things.
Shall I amend the last commit or create a new commit deleting the file?

Reply:
In your next commit you can simply uncommit or delete this file.